nservicebus/upgrades/5to6/moving-away-from-ibus.md

```diff
@@ -77,29 +77,42 @@ If the endpoint is hosted using NServiceBus.Host, use the [`IWantToRunWhenEndpoi
 
 ## Dependency Injection
 
-In previous versions the `IBus` interface was automatically registered in the IOC container. In Version 6, the new context-aware interfaces, for example, `IEndpointInstance`, `IMessageSession` and `IMessageHandlerContext`, etc., will not be automatically registered in the [Container](/nservicebus/containers/).
+In previous versions, the `IBus` interface was automatically registered in the IOC container. In Version 6, the new context-aware interfaces, for example, `IEndpointInstance`, `IMessageSession` and `IMessageHandlerContext`, etc., will not be automatically registered in the [Container](/nservicebus/containers/).
```

Comment:
Shouldn't it be: "are not be automatically", instead of "will not be automatically"